modules/customizing-the-appearance/proc-customize-rhdh-sidebar-menuitems.adoc

```diff
@@ -1,129 +1,57 @@
-
-<5> This optional attribute specifies the parent menu item under which the current item is nested. If this attribute is used, the parent menu item must be defined elsewhere in the `menuItems` configuration of any enabled plugin. You can define this attribute for each section.
-
-.Example `menuItems` configuration
 [source,yaml]
 ----
 dynamicPlugins:
   frontend:
-    <package_name>:
-      dynamicRoutes:
-        - path: /my-plugin
-          module: CustomModule
-          importName: FooPluginPage
-          menuItem:
-            icon: fooIcon
-            text: Foo Plugin Page
+    default.main-menu-items:
       menuItems:
-        my-plugin: # matches `path` in `dynamicRoutes`
-          priority: 10 # controls order of plugins under the parent menu item
-          parent: favorites # nests this plugin under the `favorites` parent menu item
-        favorites: # configuration for the parent menu item
-          icon: favorite # icon from RHDH system icons
-          title: Favorites # title for the parent menu item
-          priority: 100 # controls the order of this top-level menu item
-----
---
-
-. To ensure that a menu item is identified as a main menu item, you must add the `default.` prefix to its key. For example:
-+
---
-.Example configuration of main menu items in sidebar navigation
-[source,yaml]
-----
-dynamicPlugins:
-  frontend:
-    default.main-menu-items: # key for configuring static main menu items
-      menuItems: 
-        default.<menu_item_name>: # key of the menu item configuration. `default.` prefix is required for a main menu item key <1>
-          parent: my_menu_group # optional, specifies the parent menu item for this item
-          priority: 10 # optional, specifies the order of this menu item within its menu level
-        default.<menu_group_parent_item_name>: # must be configured if it is specified as the parent of any menu items. `default.` prefix is required for a main group parent item key <1>
-          icon: my_menu_group_icon # required for parent menu items, defines the icon for the menu group
-          title: my_menu_group_title # required for parent menu items, defines the icon for the menu group
-          priority: 100 # optional, specifies the order of the parent menu item in the sidebar
-----
-
-
-<1> The `default.` prefix identifies an item as a main menu item. You can add the `default.` prefix to both individual menu items or parent menu group configuration, such as `default.<menu_group_parent_item_name>` in the previous example.
-
-[NOTE]
-====
-The default priority of main menu items determines their order in the sidebar. You can customize the order of the static main menu items by adjusting their priority values. Ensure that the priority and title of each item is clear to facilitate easy reordering. 
-====
---
-
-
-
-
-
-
-
+        default._<menu_item_name>_: # <1>
+          parent: _<my_menu_group>_ # <2>
+          priority: 10 # <3>
```

Comment:
```suggestion
        default._<menu_item_name>_: # <1>
          parent: _<my_menu_group>_ # <2>
          priority: 10 # <3>
          icon:  # home | group | category | extension | school | _<your_icon>_ #
          title:  _<my_menu_title_>
```
As suggested above icon and title can be added for `default._<menu_item_name>_` so good to metion that and if this title is also set to `""` then it gets removed from the sidebar.